.github/scripts/k3d-import-retry.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+container=${1:-k3d-upstream}
+name=${2:-fleet}
+shift
+shift
+
+echo "Run k3d import with \"$*\" and retry if '$name' is not found in 'critcl images' output"
+
+i=0
+while ! ( docker exec "$container"-server-0 /bin/crictl images | grep -q "$name" ); do
```

Comment:
Why retry based on name match absence instead of checking every image individually?
`/bin/crictl inspecti "${image}" >/dev/null` may do the trick

Reply:
Interesting, but "crictl" output wasn't reliable.